### PR TITLE
Fix description

### PR DESCRIPTION
**Pull request by Fsioutas**
Fix description

### CODE REVIEW

File: R/calcCapacity.R
Comment (giannou) [inline R/calcCapacity.R:1]: this whole file can be deleted

File: R/calcElectrProd.R
Comment (giannou) [inline R/calcElectrProd.R:1]: this function can also be deleted

File: R/calcPOP.R [hunk 1]
```diff
@@ -1,9 +1,22 @@
-calcPOP <- function(scenario = 'SSP2') {
+#' calcPOP
+#'
+#' The SSP data filtered by pop
```
Comment (giannou) [inline R/calcPOP.R:3]: better phrase the description like this: "Derive population data based on the SSP scenarios"

File: R/calcPOP.R [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+#'
+#' @param scenario string. By choosing a scenario you filter the SSP dataset
```
Comment (giannou) [inline R/calcPOP.R:5]: better list which values the argument `scenario` can take ("SSP1", "SSP2", etc.) and phrase the argument description like this: "SSP scenario choice"

**Review by giannou — CHANGES_REQUESTED** [review]

File: R/calcCapacity.R
Comment (Fsioutas) [reply R/calcCapacity.R:1]: ok

File: R/calcElectrProd.R
Comment (Fsioutas) [reply R/calcElectrProd.R:1]: ok

File: R/calcPOP.R (hunk 1 above)
Comment (Fsioutas) [reply R/calcPOP.R]: ok

File: R/calcPOP.R (hunk 2 above)
Comment (Fsioutas) [reply R/calcPOP.R]: I itemize the scenario, is it ok?

**Review by giannou — APPROVED** [review]

File: R/calcPOP.R (hunk 2 above)
Comment (giannou) [reply R/calcPOP.R]: perfect, thanks

**Pull request merged**